### PR TITLE
drawing app

### PR DESCRIPTION
**Pull request by TidbitsJS**
Closes: #116 

### Describe the changes you've made
A simple yet elegant drawing app with three - colors & brushes with an eraser. Created with canvas 

### Describe if there is any unusual behavior of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
draw

### CODE REVIEW

File: assets/css/draw.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
+html {
+    margin: 0;
+    padding: 0;
+  }
+  
+  body {
+    height: 100vh;
+    width: 100vw;
+    overflow-y: hidden;
```
Comment (Vishal-raj-1) [inline assets/css/draw.css:9]: You can simply write it `overflow: hidden` .

File: assets/css/draw.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  .size button {
+    width: 70px;
+    padding: 10px 5px;
+    margin: 5px;
+    border-radius: 5px;
+    border: 1px solid black;
+  }
+  
+  .s-small:hover {
+    transform: scale(0.85);
```
Comment (Vishal-raj-1) [inline assets/css/draw.css:64]: I think it should be 
```suggestion
    transform: scale(0.8);
```

File: assets/js/draw.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
+const ctx = document.getElementById('canvas').getContext('2d')
```
Comment (Vishal-raj-1) [inline assets/js/draw.js:1]: You should use `;` at the end of line.

File: assets/css/draw.css (hunk 1 above)
Comment (TidbitsJS) [reply assets/css/draw.css:9]: Yeah, that can also be done. Should I change that right away?

**Review by Vishal-raj-1 — COMMENTED** [review]
@TidbitsJS This projects seems cool to me. I would like to use this before merging. So, Could you please enable github pages on your forked repo so that you can share your local website. 

Check my comments and try to improve it !! Rest is awesome.

File: assets/css/draw.css (hunk 1 above)
Comment (Vishal-raj-1) [reply assets/css/draw.css:9]: Yes !!

File: assets/js/draw.js (hunk 3 above)
Comment (TidbitsJS) [reply assets/js/draw.js:1]: Here is the link - https://tidbitsjs.github.io/Awesome-JavaScript-Projects/

Okay, I will take care of this. ( Semicolons ruins life :) ... Just kidding, I'm not good with ; )

**Comment by Vishal-raj-1** [comment]
Looking awesome for me !! I am merging it. @TidbitsJS I have some suggestion Like `clear all` so that we can remove everything and make a new page. You can open the issue for the same. Any more suggestions are always welcome.

Thanks for your contribution. Lookiing forward to more PR by your side.

**Comment by TidbitsJS** [comment]
> Looking awesome for me !! I am merging it. @TidbitsJS I have some suggestion Like `clear all` so that we can remove everything and make a new page. You can open the issue for the same. Any more suggestions are always welcome.
> 
> Thanks for your contribution. Lookiing forward to more PR by your side.

That's really a good suggestion @Vishal-raj-1. I will do that soon. 

Thank you for supporting. & yeah many more PR's are on the way

**Pull request merged**